src/NATS.Client.JetStream/Models/ConsumerCreateRequestAction.cs

```diff
@@ -0,0 +1,11 @@
+using System.Runtime.Serialization;
+
+namespace NATS.Client.JetStream.Models;
+
+public enum ConsumerCreateRequestAction
```

Comment:
what do you think of `ConsumerCreateAction` as the name? more succinct?

Reply:
Yes, think it's good too. Renamed